server/tests/backends/sql_translator_integration_tests/test_sql_redshift_translator_steps.py

```diff
@@ -1,162 +1,76 @@
 import json
-import time
 from os import environ
-from random import randint
-from typing import Any
 
 import pandas as pd
 import pytest
 import redshift_connector
-import sqlalchemy
-from sqlalchemy.engine.url import URL
+from redshift_connector.core import Connection
 
 from tests.utils import assert_dataframes_equals, get_spec_from_json_fixture, retrieve_case
-from weaverbird.backends.pandas_executor.pipeline_executor import logger
 from weaverbird.backends.pypika_translator.dialects import SQLDialect
 from weaverbird.backends.pypika_translator.translate import translate_pipeline
-from weaverbird.pipeline import PipelineWithVariables
+from weaverbird.pipeline import Pipeline
+
+_HOST = 'toucan-paris.crxviwjnhzks.eu-west-3.redshift.amazonaws.com'
+_CLUSTER = 'toucan-paris'
+_USER = 'awsuser'
+_DATABASE = 'dev'
+_PASSWORD = environ.get('REDSHIFT_PASSWORD')
+_REGION = "eu-west-3"
+_PORT = 5439
+
+
+@pytest.fixture
+def redshift_connection() -> Connection:
+    with redshift_connector.connect(
+        user=_USER,
+        database=_DATABASE,
+        host=_HOST,
+        cluster_identifier=_CLUSTER,
+        password=_PASSWORD,
+        region=_REGION,
+        port=_PORT,
+        ssl=False,
+    ) as connection:
+        connection.autocommit = True
+
+    return connection
+
+
+_BEERS_TABLE_COLUMNS = [
+    "price_per_l",
+    "alcohol_degree",
+    "name",
+    "cost",
+    "beer_kind",
+    "volume_ml",
+    "brewing_date",
+    "nullable_name",
+]
```

Comment:
Have you been able to insert the dataset into redshift ?

Reply:
not yet, it's skipped it's a subject of a card i created here : https://toucantoco.atlassian.net/jira/software/c/projects/TCTC/boards/18/backlog?view=detail&selectedIssue=TCTC-3204&issueLimit=100